.eslintrc.json

```diff
@@ -45,6 +52,151 @@
         "allowTemplateLiterals": true,
         "avoidEscape": true
```

Comment:
Remove the rules `eqeqeq`, `linebreak-style`, `no-unused-vars` and `quotes`. Eslint and prettier come with sensible defaults already, it's just overhead to define our own.